src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/pkg/PackageManager.kt

```diff
@@ -286,8 +305,23 @@ class PackageManager(sync: InstanceSync) : InstanceService(sync) {
             logger.info("Installing package '$remotePath' on $instance")
 
             val response = try {
-                http.postMultipart(url, mapOf("recursive" to installRecursive.get())) {
-                    InstallResponse.from(asStream(it), responseBuffer.get())
+                http.postMultipart(url, mapOf("recursive" to installRecursive.get())) { response ->
+                    if (responseDumping.get()) {
+                        val dumpFile = responseDir.map { it.dir("$it/${instance.name}") }
+                                .get().asFile
+                                .resolve(remotePath.removePrefix("/"))
+                                .apply {
+                                    delete()
+                                    parentFile.mkdirs()
+                                }
+                                .run { parentFile.resolve("$nameWithoutExtension.html") }
+                        logger.info("Dumping package installation response to file '$dumpFile'")
+                        val teeOut = dumpFile.outputStream()
```

Comment:
as of pkg deploy is usually done in parallel logging to console is not an option (possible with capturing whole responses which could be big); instead, last pkg deploy installation response is always saved in tmp dir, and could be easily reviewed :)